tests/test_config_templates/test_sync_script.py

```
@@ -0,0 +1,213 @@
+    dest = tmp_path / "dest"
+    dest.mkdir()
+
+    # Simulate the sync operation from sync.sh lines 225-240
+    # This is the buggy behavior - it copies everything recursively
+    src_path = source / ".github"
+    dest_path = dest / ".github"
+    dest_path.mkdir(parents=True)
+
+    logger.info("Copying directory recursively (simulating buggy behavior): %s -> %s", src_path, dest_path)
+    # This is what the current script does (copies everything)
+    subprocess.run(["cp", "-R", f"{src_path}/.", f"{dest_path}/"], check=True)
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)